File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {         
    nums1 = nums1.concat(nums2);
    nums1.sort((a,b) => a - b);
    let rem = nums1.length%2;
    let i = Math.floor(nums1.length/2);
    if(rem==0){
        return (nums1[i]+nums1[i-1])/2;
    } else{
        return (nums1[i]);
    }
};

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int arraySize = nums1.length + nums2.length;
        int[] newArray = new int[arraySize];
        System.arraycopy(nums1, 0, newArray, 0, nums1.length);
        System.arraycopy(nums2, 0, newArray, nums1.length, nums2.length);
        Arrays.sort(newArray);
        boolean odd = arraySize%2 != 0;
        double result;
        int mid = arraySize / 2;
        if(odd){
            result =  newArray[mid];
        } else{
            result = (newArray[mid-1] + newArray[mid])/2.0;
        }
        return result;
    }
}